EmpleoDotNet.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
+# Visual Studio 2013
+VisualStudioVersion = 12.0.31101.0
```

Comment:
This guy suggest having two .sln files http://stackoverflow.com/questions/32439863/how-to-stop-visual-studio-2013-to-change-sln-version is it really a good idea?